app/ts/components/pages/SettingsView.tsx

```diff
@@ -44,7 +45,10 @@ function ImportExport() {
 		}
 		browser.runtime.onMessage.addListener(popupMessageListener)
 
-		return () => browser.runtime.onMessage.removeListener(popupMessageListener)
+		return () => { () => {
+			browser.runtime.onMessage.removeListener(popupMessageListener)
+			checkAndPrintRuntimeLastError()
+		} }
```

Comment:
`() => void` is also a valid useEffect callback

```suggestion
		return () => {
			browser.runtime.onMessage.removeListener(popupMessageListener)
			checkAndPrintRuntimeLastError()
		}
```